<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>receive</class>
 <widget class="QWidget" name="receive">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QCheckBox::indicator:checked  {image: url(:/icon/check_white.png);}  QCheckBox::indicator:unchecked  { image: url(:/icon/uncheck.png);  }
QCheckBox::indicator{width: 23px;height:23px;}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>29</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="horizontalSpacing">
        <number>17</number>
       </property>
       <property name="verticalSpacing">
        <number>14</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="label_amount">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>18</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>18</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
color: #ffffff;</string>
         </property>
         <property name="text">
          <string>AMOUNT</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_message">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>18</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>18</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
color: #ffffff;</string>
         </property>
         <property name="text">
          <string>MESSAGE</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>49</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>49</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
color: #ffffff;</string>
         </property>
         <property name="text">
          <string>LABEL</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="button_request">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>49</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>49</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #b6a27e;
color: #f2f2f2;</string>
         </property>
         <property name="text">
          <string>REQUEST</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>7</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="check_resuse_address">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>23</width>
             <height>23</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>23</width>
             <height>23</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: transparent</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>23</width>
             <height>23</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_reuse_address">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>18</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>18</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color: #f2f2f2;
background-color: transparent;</string>
           </property>
           <property name="text">
            <string>REUSE EXISTING ADDRESS </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QWidget" name="widget_label" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>467</width>
           <height>49</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>467</width>
           <height>49</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #29333d;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_3"/>
       </item>
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="edit_amount_value">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>467</width>
             <height>49</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>467</width>
             <height>49</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:#29333d;
color: #ffdfb5;</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QLineEdit" name="edit_message_value">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>46</horstretch>
             <verstretch>49</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>467</width>
             <height>49</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #29333d;
color: #ffdfb5;</string>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="label_table_title">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: transparent;
padding-left: 97px;
color: #ffffff;
padding-bottom: 11px;</string>
       </property>
       <property name="text">
        <string>REQUESTED PAYMENTS HISTORY</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QWidget" name="widget_table" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: blue;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
